build/pgbadger/Dockerfile

```diff
@@ -20,11 +23,37 @@ LABEL name="pgbadger" \
 	io.k8s.display-name="pgBadger" \
 	io.openshift.tags="postgresql,postgres,monitoring,pgbadger,database,crunchy"
 
-RUN yum -y install \
-		--enablerepo="rhel-7-server-optional-rpms" \
-		--setopt=skip_missing_names_on_install=False \
-		pgbadger \
-	&& yum -y clean all --enablerepo="rhel-7-server-optional-rpms"
+RUN if [ "$DFSET" = "centos" ] ; then \
+       ${PACKAGER} -y install \
+                --enablerepo="epel" \
+                --setopt=skip_missing_names_on_install=False \
+                pgbadger \
+        && yum -y clean all ; \
```

Comment:
`yum` is called here and a few more times in this file. Should it be `${PACKAGER}`?

Reply:
Yes it should.  I will fix it.